Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -35,179 +35,127 @@ import ArcGIS
 /// example, the Toolkit's `FloatingPanel`.
 ///
 /// To see it in action, try out the [Examples](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/tree/main/Examples/Examples)
 /// and refer to
 /// [PopupExampleView.swift](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Examples/Examples/PopupExampleView.swift)
-/// in the project. To learn more about using the `PopupView` see the <doc:PopupViewTutorial>.
+/// in the project. To learn more about using the `PopupView`, see the <doc:PopupViewTutorial>.
 public struct PopupView: View {
-    /// Creates a `PopupView` with the given popup.
-    /// - Parameters:
-    ///   - popup: The popup to display.
-    ///   - isPresented: A Boolean value indicating if the view is presented.
-    public init(popup: Popup, isPresented: Binding<Bool>? = nil) {
-        self.popup = popup
-        self.isPresented = isPresented
-    }
-    
-    /// The `Popup` to display.
-    private let popup: Popup
+    /// A binding to the popup currently being presented in the navigation layer.
+    @Binding private var presentedPopup: Popup?
     
-    /// A Boolean value specifying whether a "close" button should be shown or not. If the "close"
-    /// button is shown, you should pass in the `isPresented` argument to the initializer,
-    /// so that the the "close" button can close the view.
-    var showCloseButton = false
-    
-    /// A Boolean value indicating whether the deprecated `PopupView.showCloseButton(_:)`
-    /// modifier is applied.
+    /// The popups that have been presented in the navigation layer.
     ///
-    /// Once the modifier is removed this property can be removed and the presence or lack of a value for
-    /// `isPresented` should be the sole criteria to show or hide the close button.
-    var showCloseButtonDeprecatedModifierIsApplied = false
+    /// There is one popup for every `InternalPopupView` in the current
+    /// navigation stack.
+    @State private var presentedPopups: [Popup] = []
     
-    /// The visibility of the popup header.
-    var headerVisibility: Visibility = .automatic
+    /// The model for the navigation layer.
+    @State private var navigationLayerModel: NavigationLayerModel?
     
-    /// The result of evaluating the popup expressions.
-    @State private var evaluation: Evaluation?
+    /// The visibility of the close button.
+    var closeButtonVisibility: Visibility = .automatic
     
     /// A binding to a Boolean value that determines whether the view is presented.
-    private var isPresented: Binding<Bool>?
+    ///
+    /// This property can be removed once the deprecated
+    /// `init(popup:isPresented:)` initializer is removed.
+    var isPresented: Binding<Bool>?
+    
+    /// Creates a `PopupView` to display a given popup.
+    /// - Parameters:
+    ///   - popup: A binding to the popup that view displays. When `popup` is non-`nil`, the `PopupView` is displayed.
+    /// - Since: 200.8
```

Comment:
```suggestion
    /// Creates a `PopupView` to display a given popup.
    /// - Parameter popup: A binding to the popup that view displays. When `popup` is non-`nil`, the `PopupView` is displayed.
    /// - Since: 200.8
```